components.yml

```diff
@@ -24,9 +24,10 @@ deployed_component_versions:
   greenbox:
     greenbox-cromwell: 35-0a01d91-SNAP
     greenbox-falcon: v0.1.3
-    greenbox-lira: v0.13.2
+    greenbox-lira: v0.14.0
     greenbox-pipeline-tools: v0.28.2
     greenbox-skylab:
-      smartseq2: smartseq2_v1.0.0
-  data-store: staging-2018-09-19-15-37-11.release
+      smartseq2: smartseq2_v2.0.0
+  data-store: 56da699cc785c7258d67754ec1987c945960f8c0
```

Comment:
change to `2018-10-11-15-48-22-staging.release`, i'll update the release doc

Reply:
fixed